### PR TITLE
 Add hint how to display supported file link types

### PR DESCRIPTION
**Pull request by lumbric**

### CODE REVIEW

File: content/docs/user-guide/data-management/large-dataset-optimization.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ efficiency:
 
 [replace them]: /doc/user-guide/how-to/update-tracked-files
 
+<admon type="tip">   
+Use `dvc version` in a DVC repository (with an existing cache folder) to display supported file link types.
```
Comment (dberenbaum) [inline content/docs/user-guide/data-management/large-dataset-optimization.md:102]:
```suggestion
Use `dvc doctor` in a DVC repository (with an existing cache folder) to display supported file link types.
```

They are the same, but we tend to use `dvc doctor`.

**Review by dberenbaum — APPROVED** [review]
Thanks @lumbric! LGTM although I suggested one minor change.

File: content/docs/user-guide/data-management/large-dataset-optimization.md (hunk 1 above)
Comment (lumbric) [reply content/docs/user-guide/data-management/large-dataset-optimization.md:102]: Ok thanks! In the documentation `dvc doctor` is described as alias, should this be swapped with `dvc version`, i.e. `dvc version` being an alias and `dvc doctor` the main command?

File: content/docs/user-guide/data-management/large-dataset-optimization.md [hunk 2]
```diff
@@ -98,6 +98,11 @@ efficiency:
 
 [replace them]: /doc/user-guide/how-to/update-tracked-files
 
+<admon type="tip">   
+Use `dvc version` in a DVC repository (with an existing cache folder) to display
```
Comment (shcheklein) [inline content/docs/user-guide/data-management/large-dataset-optimization.md:102]: btw, @yathomasi - for whatever reason the 80 symbols rule was not triggered within this admon section. Not critical at all. If you know from the top your had what it is about we can fix it.

Comment (yathomasi) [reply content/docs/user-guide/data-management/large-dataset-optimization.md:102]: One caveat is the multiline snippet inside a `tag` also needs to be wrapped with newlines for formatting to work properly.

Eg:
```html
<admon type="tip">   

content

</admon>
```

**Pull request merged**